app/src/main/res/layout/activity_main.xml

```diff
@@ -31,225 +30,301 @@
+        app:layout_constraintStart_toEndOf="@+id/tvOpeningBracket"
+        app:layout_constraintTop_toBottomOf="@+id/guideline1"
+         />
+
+    <TextView
+        android:id="@+id/tvDivide"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_margin="8dp"
+        app:layout_constraintDimensionRatio="1:1"
+        android:background="@drawable/round_border"
+        android:gravity="center"
+        android:text="/"
+        app:layout_constraintBottom_toTopOf="@+id/tvMultiply"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/tvClosingBracket"
+        app:layout_constraintTop_toBottomOf="@+id/guideline1"
+         />
+
+    <TextView
+        android:id="@+id/tv7"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_margin="8dp"
+        app:layout_constraintDimensionRatio="1:1"
+        android:background="@drawable/round_border"
+        android:gravity="center"
+        android:text="7"
+        app:layout_constraintBottom_toTopOf="@+id/tv4"
+        app:layout_constraintEnd_toStartOf="@+id/tv8"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/tvAC"
+         />
+
+    <TextView
+        android:id="@+id/tv8"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_margin="8dp"
+        app:layout_constraintDimensionRatio="1:1"
+        android:background="@drawable/round_border"
+        android:gravity="center"
+        android:text="8"
+        app:layout_constraintBottom_toTopOf="@+id/tv5"
+        app:layout_constraintEnd_toStartOf="@+id/tv9"
+        app:layout_constraintStart_toEndOf="@+id/tv7"
+        app:layout_constraintTop_toBottomOf="@+id/tvOpeningBracket"
+         />
+
+    <TextView
+        android:id="@+id/tv9"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_margin="8dp"
+        app:layout_constraintDimensionRatio="1:1"
+        android:background="@drawable/round_border"
+        android:gravity="center"
+        android:text="9"
+        app:layout_constraintBottom_toTopOf="@+id/tv6"
+        app:layout_constraintEnd_toStartOf="@+id/tvMultiply"
+        app:layout_constraintStart_toEndOf="@+id/tv8"
+        app:layout_constraintTop_toBottomOf="@+id/tvClosingBracket"
+         />
+
+    <TextView
+        android:id="@+id/tvMultiply"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_margin="8dp"
+        app:layout_constraintDimensionRatio="1:1"
+        android:background="@drawable/round_border"
+        android:gravity="center"
+        android:text="*"
+        app:layout_constraintBottom_toTopOf="@+id/tvSubstract"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/tv9"
+        app:layout_constraintTop_toBottomOf="@+id/tvDivide"
```

Comment:
Good to see the use of chains. Did you used right click on the views to add chains or did you added manually in the xml?